app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from ..player import Player

```

Comment:
do not use relative imports - is a better practice to use full path e.g. `from app.player...`
check in all files and fix where necessary